src/components/UserAuth/Login.css

```diff
@@ -0,0 +1,57 @@
+/*-------------------------Login page(card)---------------*/
+
+.login-check:invalid {
+	border: var(--Mak-colors-red-500) solid 1px;
+	border-radius: 1.5rem;
+}
+/* .login-check:valid {
+	border: var(--Mak-colors-green-500) solid 1px;
+	border-radius: 1.5rem;
+} */
+
+.login-check:invalid + span:before {
+	position: relative;
+	content: "✖";
+	color: var(--Mak-colors-red-500);
+	padding-left: 5px;
```

Comment:
you can use Rem values for margin paddings

Reply:
I'll make the changes.